backend/app.py

```
@@ -68,7 +73,8 @@ def get_categories():
         conn.close()
         return jsonify({"categories": categories})
     except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)